hyperspy/_signals/lazy.py

```diff
@@ -131,6 +133,33 @@ def compute(self, close_file=False, show_progressbar=None, **kwargs):
 
     compute.__doc__ %= SHOW_PROGRESSBAR_ARG
 
+    def rechunk(self,
+                nav_chunks="auto",
+                sig_chunks= -1,
+                threshold=None,
+                block_size_limit=None,
+                balance=False):
```

Comment:
`balance` can only be used for dask > 2.30, see

https://github.com/hyperspy/hyperspy/blob/165e407e777b503cdbc1b3d29dcf481ac15b7e4b/hyperspy/_signals/lazy.py#L1101-L1104

Reply:
Thanks! I just replaced it with kwargs... probably should have done that originally.